src/local/butler/bootstrap.py

```diff
@@ -18,14 +18,10 @@
 from local.butler import common
 
 
-def execute(args):
+def execute(_):
```

Comment:
I think you've said that `del X` is the preferred style these days.

Reply:
good call. fixed!